packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -218,7 +237,8 @@ function CustomViewShell(props: TCustomViewShellProps) {
                   <Route
                     path={`/custom-views/${hostContext.customViewConfig.id}/projects/${hostContext.projectKey}`}
                   >
-                    {props.children}
+                    <ModalPageTopBar onClose={handleClose} hidePathLabel />
+                    <ContentWrapper>{props.children}</ContentWrapper>
```

Comment:
I guess this results in the extra space at the top that we see here:

<img width="768" alt="image" src="https://github.com/commercetools/merchant-center-application-kit/assets/1110551/a28b4ab4-64b1-4b96-a3f0-b595dcb3cb6e">

Wondering maybe if we should define the close button as a fixed/absolute position to not affect the page layout? 🤔

Then we don't need to expose `<ModalPageTopBar>`.

Reply:
That would be an issue for notifications, which should be rendered before the close icon and push it down.

Reply:
Ok, let's maybe have a sync about it when Natalia is back and see what we can do about it.